frontend/src/pages/onboarding/index.tsx

```diff
@@ -47,15 +55,19 @@ const onboardingSteps: OnboardingStep[] = [
     subtitle: 'Other climbers will use this to identify you',
```

Comment:
Can consider center-aligning the subtitle text (such as 'Other Climbers will use this to identify you') for all onboarding pages